index.php

```diff
@@ -61,7 +63,18 @@
 
 			<!-- This class is for my footer-->	
 			<div class="footer">
-				<h1>Footer</h1>
+				<div class="footer1">
+					<h2 style="text-align: left; font-size: 1.8vw;">AUTHORS NOTE</h2>
+					<h3 style="text-align: left; font-size: 1.3vw;">© Faith Tuason, <br>Tawa College 2022, <br>all rights reserved.</h3>
+				</div>
+				<div class="footer2">
+					<img style="width: 100%; max-width: 7.3vw; padding-top: 35px;" src="images/placeholder.png" alt="logo">
```

Comment:
I used the viewport width technique on my footer images to make them more responsive